examples/Civis_WBCD_2017.ipynb

```diff
@@ -0,0 +1,676 @@
4+89dZbaq7lNzQWJJ49e/awdOlSRowYwaRJk9Rci4gkmPy2YA8AGrj7L2ZWEZgDTIpPLImX9evX\nU7VqVVq2bMnatWt1pTfJi8aCBLF161bKli1L+fLlWbx4sepVRCRB5bcP9q/u/guAu28pYF5JQq+9\n9hpnnHEG06ZNA9CHtRyJxoIE8PXXX3PBBRdwzTXXAKpXEZFElt8W7NPNbGb4vgG1cj3G3XsWtHAz\nSwP+AaQAE939wSPM1xt4DWjq7rqubxw88cQTDBs2jBYtWtC+ffug40hiK/RYIIXz6aef0rlzZ/bu\n3cstt9wSdBwRESlAfg12r8Mejz2aBZtZCvAUcAmwAfjIzGbnPo9ueL7yhC6//OHRLF+OTU5ODnfe\neSePPPIIPXr04JVXXtGWMClIocYCKZx58+bRs2dPKlSoQEZGBvXq1Qs6koiIFOCIDba7ZxRy2c2A\n1e6+FsDMXgW6A1mHzXcv8DBwRyHXJxFYsGABjzzyCEOGDOHJJ58kJSUl6EiS4KIwFsgx2rt3L1df\nfTXVq1dn7ty5pKamBh1JREQiEMml0o/VacD6XI83AOfnnsHMzgWquvsbZnbEBtvMrgeuB6hWrVoM\nohZ97o6ZcfHFF7Nw4UJatWqFmQUdS4og1Wt0uDulS5dm7ty5VKtWjQoVKgQdSUREIhTLg5Xy6t78\n0JNmxwFjgNsLWpC7T3D3Ju7epGLFilGMWDx8//33tG7dmiVLlgDQunVrNdcSM6rXwsnJyWH48OGM\nHDkSgAYNGqi5FhFJMhE32GZ2/FEuewNQNdfjVOC7XI/LA2cDC8xsHdAcmG1mTY5yPZKPr776ipYt\nW/LJJ5+wY8eOoONIEXAMY4FE6Ndff6V///6MGTOGnTt34u4Fv0hERBJOgQ22mTUzs8+B/4QfNzSz\nJyNY9kfAGWZW08xKAf2A2QefdPcd7n6Ku9dw9xrAEqCbziISPR9++CEXXHABu3btYsGCBXTs2DHo\nSJLECjEWSAR27NhBWloa06ZN4+GHH+bxxx/XN00iIkkqki3YTwBdgG0A7v5voG1BL3L3bGAo8Daw\nEpjm7l+Y2T1m1u3YI0skli9fzsUXX8wJJ5zABx98QNOmTYOOJMnvmMYCKdiBAwdo164dmZmZvPTS\nS/zpT39Scy0iksQiOcjxOHf/5rDB/kAkC3f3OYSu+pZ72sgjzNsmkmVKZOrXr89tt93GbbfdRuXK\nlYOOI0XDMY8Fkr+UlBT++Mc/UrlyZZ2XXkSkCIhkC/Z6M2sGuJmlmNkw4KsY55Jj4O6MGTOGDRs2\nkJKSwgMPPKDmWqJJY0GUZWZmkp6eDsCAAQPUXIuIFBGRNNhDgOFANeAHQgcjDollKDl62dnZ3Hjj\njQwfPpyJEycGHUeKJo0FUZSenk779u25++67OXBAXwSIiBQlBe4i4u6bCR2gKAlq9+7d9O/fn9mz\nZ/OXv/yFv//970FHkiJIY0H0jB8/nqFDh9K0aVPeeOMNXfBJRKSIKbDBNrNnyXX+6oPc/fqYJJKj\nsm3bNrp27cqSJUsYO3YsN998c9CRpIjSWFB47s7f/vY37r//frp06cLUqVMpW7Zs0LFERCTKIjnI\ncV6u+6WBy/jtFRolQCkpKWRnZzN9+nR69uwZdBwp2jQWFJKZsXv3bgYPHsz48eMpUSKWF9MVEZGg\nRLKLyNTcj83sn8D/xiyRRCQrK4vTTz+dChUqsGTJEo47LpYX5RTRWFAYu3btYuPGjZx55pk8+uij\nmJlOwyciUoQdS1dWE6ge7SASuYyMDJo3b87tt4euMq/mWgKisSACmzdvpm3btlxyySXs2bOH4447\nTs21iEgRF8k+2Nv5v/0ujwN+BO6MZSg5silTpnD11Vdz5plnctdddwUdR4oRjQVHb82aNaSlpbFx\n40ZeffVVypQpE3QkERGJg3wbbAttZmkIbAxPynH33x3kJPHx2GOPcccdd9C6dWtmzZpFhQoVgo4k\nxYTGgqO3bNkyOnfuTE5ODhkZGbRo0SLoSCIiEif57lsQ/gBNd/cD4Zs+UAOyefNmHnjgAfr06cPb\nb7+t5lriSmPB0XvggQcoV64cmZmZaq5FRIqZSA5hX2pmjd39k5inkd/Zv38/JUqUoFKlSixdupSa\nNWtqn2sJisaCCOzfv5+SJUvywgsvsGvXLk499dSgI4mISJwdsVMzs4PN94WEPlhXmdknZvapmekD\nNg527NhBx44duffeewGoVauWmmuJO40FkXF3Ro8eTevWrfnll18oX768mmsRkWIqvy3YS4HGQI84\nZZFcvvvuOzp16kRWVhbXXHNN0HGkeNNYUIADBw5w22238dRTT/Hf//3flCxZMuhIIiISoPwabANw\n9zVxyiJhK1euJC0tjR9//JE333yTDh06BB1JijeNBfnYu3cvAwYMYObMmdxxxx089NBD+qZJRKSY\ny6/Brmhmw4/0pLv/vxjkKfZ+/vln2rRpA8DChQtp3LhxsIFEojwW7NwJtWoVPlSiuOGGG5g5cyZj\nxoxh2LBhQccREZEEkF+DnQL8F+GtVxIf5cuX58knn6RJkyacfvrpQccRgSiOBe7w7bfQowjtbDJy\n5Ei6du1K7969g44iIiIJIr8Ge5O73xO3JMXcM888Q+XKlenRowd9+/YNOo5IblEbC7ZsgV9/hWrV\norG04KxYsYIXX3yRhx56iFq1alGrKG2SFxGRQstvR0FtuY4Dd2fkyJHceOONvPzyy0HHEclL1MaC\nDRtCP1NTo7XE+Fu4cCEXXnghL730Et99913QcUREJAHl12C3i1uKYio7O5vBgwdz7733cu211zJl\nypSgI4nkJWpjwb59oZ+lS0drifE1ffp0OnToQJUqVVi8eDGnnXZa0JFERCQBHbHBdvcf4xmkuNm3\nbx/du3dn0qRJjBw5kokTJ1KiRCTX/RGJr2iOB
```

Comment:
This is less common using CivisML, since cross-validation is automatic. I think this section would be a bit clearer if it referenced scoring new, unlabeled data.